### ISSUE TITLE
Geometry Dash wont let me create and is unstable

### ISSUE THREAD

**Issue by zerogrqviT** [WEGFan/Geometry-Dash-Savefile-Fix#21]
![Screenshot 2023-05-27 210230](https://github.com/WEGFan/Geometry-Dash-Savefile-Fix/assets/134754828/42b61aff-6df6-4fce-89eb-cf719b5843b9)
My game wouldn't launch so I used this program it said it fixed every file and everything worked (thank you) except it wouldn't let me make anything in the editor and crashes when I try to save a level. And now whenever I use the program the message "Traceback (most recent call last):
  File "GDSavefileFix.py", line 61, in mainzlib.error: Error -3 while decompressing data: invalid block type"
 pops up for either CC.Local_Levels or Local levels2. Another Issue that is happening is that the game always crashes when I tab out. I've uninstalled the game 2 times and reinstalled it but the same problems keep happening. Thanks for help!

**Comment by zerogrqviT** [comment]
Found out how to fix "Traceback (most recent call last): File "GDSavefileFix.py", line 61, in mainzlib.error: Error -3 while decompressing data: invalid block type" and "CC.LocalLevels.dat not found!" And now the game allows me to make and edit levels and Geometry Dash is stable. Here's what I did
1. I had went to the most recent backup that had CC.LocalLevels.dat in it (the file would delete itself if I launched the game)
2. I copy and pasted the file onto where .dat files are stored
3. I launched the game but I did nothing and exited out
4. I used the program and it gave me the same error message, but I launched Geometry Dash and the game works perfectly now!
Thank you for this amazing program!
